### DISCUSSION TITLE
missing  libjava.so in lasticsearch (Macbook M1 chip)

### DISCUSSION THREAD

**Discussion by verkaz** [markshust/docker-magento#475] [Q&A]
Hi,

I recently went over from Macbook Intel to M1 (Arm) chip and when I compose up the same file  but on my new M1 macbook I get the eleasticsearch fails to start and returns:

Error: could not find libjava.so
Error: Could not find Java SE Runtime Environment.

I am using old image: markoshust/magento-elasticsearch:6.5.4-0 for Magento 2.1.15
I appreciate your advice

**Comment by rangerz** [comment]
It's possible work for M1.
https://github.com/luancschmitz/docker-magento-mac-m1
